### DISCUSSION TITLE
Add a color accent or an animation for the system tray icon when the connection is active

### DISCUSSION THREAD

**Discussion by GregFedy** [hiddify/hiddify-app#591] [Ideas]
It is impossible to determine if the connection is active by the icon in the system tray (Windows 11).
![image](https://github.com/hiddify/hiddify-next/assets/86961366/e2ccbb44-06a2-4ba0-a9d0-368dbec843a4)
You have to click on the right mouse button to see a check mark indicating that the connection is active
![image](https://github.com/hiddify/hiddify-next/assets/86961366/1ee37b5d-b19f-44a2-af6f-35184461aaab)

For example, when the connection is active, the icon should change color from white to orange (as the main color of the application)

**Comment by Po-temkin** [comment]
This issue was fixed in pre-release version 2.1.5
